### ISSUE TITLE
The option to restore the initial mouse location when finished doesn't work on a secondary display.

### ISSUE THREAD

**Issue by alin89c** [BlueM/cliclick#119]
The option to restore the initial mouse location when finished doesn't work on a secondary display.
Am I the only one having this issue?

Thank you.

**Comment by alin89c** [comment]
While trying to find a workaround, I got this error.
![09 - 7](https://user-images.githubusercontent.com/42713989/110145097-be5fe000-7de1-11eb-885c-aefdcb1795b1.gif)

**Comment by alin89c** [comment]
This would be a workaround, but the third shell script gives this error:
`
error "Invalid Y axis coordinate “362
” given" number 1
`

Applescript (alternative to the -r option):
``` applescript
set delayOne to 1

-- copy the current mouse position to the clipboard
do shell script "eval $(/usr/libexec/path_helper -s); cliclick -d clipboard p:."
delay delayOne

-- move mouse pointer to new coordinates
do shell script "eval $(/usr/libexec/path_helper -s); cliclick m:498,51"
delay delayOne

-- click at the initial mouse pointer location
do shell script "eval $(/usr/libexec/path_helper -s); cliclick c:" & (the clipboard)
delay delayOne
```

**Comment by BlueM** [comment]
Do I assume correctly that your secondary display is arranged to the left your primary display?

**Comment by alin89c** [comment]
Yes, but I've found a workaround. To recall the initial mouse coordinates in my script (AppleScript), I just store the x and y before executing a cliclick command. Then, at the end of the script I use the move command to put the mouse pointer where it was initially.

**Comment by BlueM** [comment]
Fixed in 5.0.1, which will be released in a few minutes. Given that the fix required inserting two “=” into the code, it took me quite a bit …
